tests/test_device_config.py

```diff
@@ -119,6 +119,7 @@
         vol.Optional("format"): [FORMAT_SCHEMA],
         vol.Optional("mask"): str,
         vol.Optional("endianness"): vol.In(["little"]),
+        vol.Optional("signed_bits"): int,
```

Comment:
The test should also change to `vol.Optional("mask_signed"): True` (only true is considered valid, as it is better to be omitted in the false case).